src/http.c

```diff
@@ -99,10 +108,23 @@ int HttpSend(HttpReq *req, const void *payload, size_t sz_payload) {
                            (const char *)req->port)))
       return ret;
 
-  while (-1 == (ret = send(req->conn->fd, payload, sz_payload, MSG_DONTWAIT)))
+  while (-1 ==
+         (ret = send(req->conn->fd, payload, sz_payload,
+                     MSG_DONTWAIT | MSG_NOSIGNAL)))
```

Comment:
`MSG_NOSIGNAL` isn't supported on Mac.

Reply:
Should be fixed in 97b2d65, but haven't had a chance to check yet.